visual_behavior/decoding_population/anova_tukey_fn.py

```diff
@@ -0,0 +1,126 @@
+# Perform Anova (1 way) and pairwise Tukey HSD
+
+import numpy as np
+import pandas as pd
+import statsmodels.api as sm
+from statsmodels.formula.api import ols
+from statsmodels.stats.multicomp import (pairwise_tukeyhsd, MultiComparison)
+
+def anova_tukey(svm_df, values_stat, label_stat='experience_levels', cres=['Slc17a7', 'Sst', 'Vip'], exp_level_all=['Familiar', 'Novel 1', 'Novel >1']):
```

Comment:
Instead of using multiple # to make the top comment, use ``` <stuff> ```, that way it gets parsed by automated documentation viewers.

Reply:
not sure i'm following you here. Are you saying that instead of using # to make comments, I should use something else? I sometimes use """ or ''' .... but seems like you wrote something else? Can you clarify?

Reply:
Its better to use:
```
def example():
   ` ` `
        This is a multiline
        comment
    ` ` `
```    
Than
```
def example():
   # This is a multiline
   # comment
```

Reply:
I see my original comment is confusing because github's markdown rendered my ``` as a code block.